test/JDBC/input/functions/format-vu-verify.sql

```diff
@@ -0,0 +1,95 @@
+select FORMAT(d, 'd','en-us') from date_testing;
```

Comment:
One of the main purposes of MVU test is to verify dependent objects such as views functions and procedures. I would suggest to add at least the followings:
- `create view format_v1 as select FORMAT(...), ...`
- `create function format_f1 (... default FORMAT(...))...` (if possible)